packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -1565,6 +1566,17 @@ class PopupMenuButtonState<T> extends State<PopupMenuButton<T>> {
           child: widget.child,
         ),
       );
+      final MaterialTapTargetSize tapTargetSize = widget.style?.tapTargetSize ?? MaterialTapTargetSize.shrinkWrap;
```

Comment:
I think here we can default to the `ThemeData.materialTapTargetSize` value. We can also add this property to `PopupMenuButton` so developers can control the button size easier:) Then here would be `final MaterialTapTargetSize tapTargetSize = widget.style?.tapTargetSize ?? popupMenuTheme.tapTargetSize ?? Theme.of(context).materialTapTargetSize;`
```suggestion
      final MaterialTapTargetSize tapTargetSize = widget.style?.tapTargetSize ?? Theme.of(context).materialTapTargetSize;
```

Reply:
I tried this but

if it defaults to  Theme.of(context).materialTapTargetSize, this PR will be a breaking change because some buttons' size will be increased to 48X48, looks like some flutter_cacoon tests are failing because of that.(some g3 tests too.)

since I can't apply a flutter_cacoon fix together with this PR. I'm thinking 
1.  Merge this PR with default to MaterialTapTargetSize.shrinkWrap so it's not a breaking change.
2. Update flutter_cacoon tests and g3 tests to use MaterialTapTargetSize.padded in its PopupMenuButton
3. Update PopupMenuButton to default to Theme.of(context).materialTapTargetSize
4. Remove now redundant MaterialTapTargetSize.padded in flutter_cacoon tests and g3 tests

Reply:
I see. Sounds good to me! If we want to add this property to PopupMenuTheme, we can also add a separate PR later. This PR LGTM!